figures/settings.py

```diff
@@ -15,11 +15,20 @@
 
 DAILY_METRICS_CELERY_TASK_LABEL = 'figures-populate-daily-metrics'
 
+DEFAULT_IS_MULTISITE = False
 DEFAULT_LOG_PIPELINE_ERRORS_TO_DB = True
 
 env_tokens = {}
 
 
+def is_multisite():
+    """
+    Override by setting ``IS_MULTISITE`` to true in the Figures env settings
+    """
+    return bool(env_tokens.get('IS_MULTISITE',
```

Comment:
Since this method is not updating `figures.settings.env_tokens`, the update you suggest will work. I'll do this

Reply:
@OmarIthawi Are you suggesting doing a lazy import within the `is_multisite` function?

Reply:
@OmarIthawi Bump on this question, please.

Reply:
@johnbaldwin I think the idea of a plugin patching `settings` is very buggy and involves guessing on which module was loaded first. Instead, whenever possible, `settings` should be loaded as late as reasonably possible, and only read and not written.

What I'm suggesting is not just lazy loading. It is lazy loading to avoid Django getting confused with partially initialized settings, since the `settings` also imports a function from Figures.

 - https://github.com/appsembler/edx-platform/blob/bd259531edaec0e8824349ec1a7d20e75317e3e2/lms/envs/aws_appsembler.py#L155-L161

I'm also suggesting avoiding writing on `settings` and `ENV_TOKENS` except when absolutely needed.

There's a notable exception for the above which is the Hawthorn plugin system, but that would be out of the scope of this PR.

Please let me know if you have any questions.